### PR TITLE
Adapt tests to sci420

### PR DESCRIPTION
**Pull request by Ekopalypse**
Make tests run with scintilla 4.2.0

### CODE REVIEW

**Comment by Ekopalypse** [comment]
Is there a problem with this one?
The already merged commit, the one for the samples, shouldn't appear here, shouldn't it?
Should I have created two branches derived from the same master branch?

**Comment by chcg** [comment]
@Ekopalypse Let's see how smart the git merge is.

Just didn't merge because of some questions regarding the changes, but forgot to click submit for my review comments.

File: PythonScript/python_tests/tests/ScintillaWrapperTestCase.py [hunk 1]
```diff
@@ -801,7 +804,7 @@ def test_scintillawrapper_colour_int_void_in_callback(self):
         self.assertEqual(self.callbackCalled, True)
 
     def test_scintillawrapper_bool_int_void(self):
-        original = editor.getMarginSensitiveN(1)
+        _ = editor.getMarginSensitiveN(1)
```
[the remark this answers is not in the document]
Comment (Ekopalypse) [reply PythonScript/python_tests/tests/ScintillaWrapperTestCase.py]: Need, depends, this and the other change is basically to satisfy the python linter.

File: PythonScript/python_tests/tests/ScintillaWrapperTestCase.py [hunk 2]
```diff
@@ -2,7 +2,10 @@
 import unittest
 import time
 
-from Npp import *
+from Npp import (
```
[the remark this answers is not in the document]
Comment (Ekopalypse) [reply PythonScript/python_tests/tests/ScintillaWrapperTestCase.py]: >Why are the imports in braces?
>What is the benefit to import just the necessary module parts for the tests?
In this case, both are Linter issues.

**Pull request merged**